README.md

```diff
@@ -59,6 +59,11 @@ Finally, run the following command from inside of the application directory:
 ng serve --open
 ```
 
+Alternatively you can run the following command from the root directory where your index.html is located:
```

Comment:
Feel free to remove this since it's not related to the component selector issue.

Reply:
This will only work if you run `ng build` before doing `npx light-server`, so I think we should move this into the Building section.